test/gc/test-net-timeout.js

```diff
@@ -35,7 +35,11 @@ function getall() {
   if (count >= todo)
     return;
 
-  const req = net.connect(server.address().port, server.address().address);
+  if (common.hasIPv6)
+   var host = '::1';
+  else
+    var host = '127.0.0.1';
```

Comment:
Can you either use `common.localhostIPv4` here or (more succinctly) make the server listen on that address?